system/ee/language/english/content_lang.php

```diff
@@ -1013,6 +1013,10 @@
 
     'entry_manager_view_not_updated_desc' => 'We were unable to update this view, please review and fix errors below.',
 
+    'icon_expiration_date' => 'Icon Expiration Date',
+
+    'column_icon_entry_date' => 'Icon Entry Date',
+
```

Comment:
I assume this is not needed anymore?

Reply:
No, you're right. Good find. Will remove.